### ISSUE TITLE
How can I feed MNIST net  to vl_simplenn for testing 

### ISSUE THREAD

**Issue by DoDNet** [vlfeat/matconvnet#291]
Hi,
I am new in CNNs. I am trying to get used to MatConvNet by using the provided examples. Basically, I run the MNIST training example and got the multiple 'net-epoch-i.mat' files. Then I tried to test 'net-epoch-20.mat' on a new image 'digit.png' using vl_simplenn(net, im_). I expected getting an array  'res(end).x' of 10 values however I get a 'res(end).x' of '50x50x10' values. Please, can you tell me what should I change in the training code to get the right net to be able to test it later using vl_simplenn on a new image. 

I execute the next script:

run ..\..\matlab\vl_setupnn
net = load('net-epoch-20.mat');
net.net.layers{end}.type='softmax'
im = imread('digit.png')
a = single(im) -  mean(im(:));
im_ = imresize(a, [224 224]);
res = vl_simplenn(net.net, im_) ; 

Best regards
